sources/packages/web/src/components/aest/StudentApplications.vue

```diff
@@ -159,7 +159,7 @@ export default {
       default: false,
     },
   },
-  setup(props: any) {
+  setup(props: any, { emit }) {
```

Comment:
can use defineComponent and type the props

Reply:
I belive we are planning to have a ticket for it. but i will update it